Buildings/Fluid/ZoneEquipment/FanCoilUnit/Controls/ConstantFan_VariableWaterFlowrate.mo

```diff
@@ -0,0 +1,176 @@
+  "Controller for fan coil system with variable water flow rates and fixed speed fan"
+
+  parameter .Buildings.Controls.OBC.CDL.Types.SimpleController controllerTypeCoo=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller" annotation (Dialog(group="Cooling mode control"));
+  parameter Real kCoo=1 "Gain of controller"
+    annotation(Dialog(group="Cooling mode control"));
+  parameter Real TiCoo=0.5 "Time constant of integrator block"
+    annotation(Dialog(group="Cooling mode control",
+      enable = controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PI or
+      controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Real TdCoo=0.1 "Time constant of derivative block"
+    annotation(Dialog(group="Cooling mode control",
+      enable = controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PD or
+      controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+
+  parameter .Buildings.Controls.OBC.CDL.Types.SimpleController controllerTypeHea=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller" annotation (Dialog(group="Heating mode control"));
+
+  parameter Real kHea=1 "Gain of controller"
+    annotation(Dialog(group="Heating mode control"));
+  parameter Real TiHea=0.5 "Time constant of integrator block"
+    annotation(Dialog(group="Heating mode control",
+      enable = controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PI or
+      controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Real TdHea=0.1 "Time constant of derivative block"
+    annotation(Dialog(group="Heating mode control",
+      enable = controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PD or
+      controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+
+  parameter Real dTHys(
+    final unit="K",
+    displayUnit="K",
+    final quantity="TemperatureDifference") = 0.2
+    "Temperature difference used for enabling coooling and heating mode"
+    annotation(Dialog(tab="Advanced"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TZon "Measured zone temperature"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,20},{-100,60}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TCooSet
+    "Zone cooling temperature setpoint"
+    annotation (Placement(transformation(extent={{-140,0},{-100,40}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput THeaSet
+    "Zone heating temperature setpoint"
+    annotation (Placement(transformation(extent={{-140,-40},{-100,0}}),
+        iconTransformation(extent={{-140,-60},{-100,-20}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yFan "Fan enable signal"
+                                                           annotation (Placement(
```

Comment:
consider to change it into "onFan"?

Reply:
I am using the OBC convention. I will change it to onFan on the system model